runtime/tutorials/how_to_with_npm/prisma.md

```diff
@@ -53,39 +53,33 @@ model Dinosaur {
 }
 ```
 
-Prisma should also have generated a `.env` file with `DATABASE_URL`. Let's
-assign `DATABASE_URL` to a PostreSQL connection string. In this example, we'll
+Prisma also have generates a `.env` file with `DATABASE_URL`. Let's
+assign `DATABASE_URL` to a PostgreSQL connection string. In this example, we'll
 use a free [PostgreSQL database from Supabase](https://supabase.com/database).
 
 Next, let's create the database schema:
 
 ```shell, ignore
-deno run -A npm:prisma@^4.5 db push
+deno run -A npm:prisma@latest db push
 ```
 
-After that's complete, we'll need to generate a Prisma client for Data Proxy:
+After that's complete, we'll need to generate a Prisma Client:
 
 ```shell, ignore
-deno run -A --unstable npm:prisma@^4.5 generate --data-proxy
+deno run -A --unstable npm:prisma@latest generate --no-engine
 ```
 
-## Setup Prisma Data Platform
+## Setup Accelerate in the Prisma Data Platform
 
-In order to use Prisma Data Platform, we'll have to create and connect a GitHub
-repo. So let's initialize the repository, create a new GitHub repo, add the
-remote origin, and push the repo.
+To get started with the Prisma Data Platform:
 
-Next, sign up for a free
-[Prisma Data Platform account](https://cloud.prisma.io/).
+1. Sign up for a free [Prisma Data Platform account](https://console.prisma.io)
+2. Create a project
+3. Navigate to the project you created
+4. Enable Accelerate
+5. Generate an Accelerate connection string and copy it to your clipboard.
```

Comment:
This doesn't mention that you need to enter the original connection string for Postgres anywhere.